nautobot_chatops/tests/test_dispatchers.py

```diff
@@ -119,6 +119,22 @@ def test_get_prompt_from_menu_choices(self):
         response = self.dispatcher.get_prompt_from_menu_choices(choices, offset=500)
         self.assertEqual(response, expected_choices)
 
+    @patch("nautobot_chatops.dispatchers.slack.SlackDispatcher.send_snippet")
+    def test_send_snippet_no_title(self, mock_send_snippet):
+        """Make sure send_snippet can be called with no title."""
+        self.dispatcher.send_snippet("Testing snippet.")
+        args = mock_send_snippet.call_args[0]
+        self.assertTrue(mock_send_snippet.called)
+        self.assertEqual(args, ("Testing snippet.",))
```

Comment:
This has been updated. Thanks Glenn! My build is failing but it has something to do with docker requests. I'm guessing because I had two commits really close together.